src/test/java/stepDefinition/ContactUsSteps.java

```diff
@@ -23,4 +24,23 @@ public void userValidatesThatContactUsPageIsOpen() {
         WebElement contactUsPage = driver.findElement(ContactUsPage.ContactUsHeading);
         Assert.assertTrue(contactUsPage.isDisplayed());
     }
+
+    @When("User clicks on input field {string} and enters {string}")
+    public void userClicksOnInputFieldNameAndEntersGuestName(String placeHolder, String info) {
+        WebElement inputGuestInfo = driver.findElement(ContactUsPage.sendMessageFormInput(placeHolder));
+        SeleniumActions.click(ContactUsPage.sendMessageFormInput(placeHolder), driver);
+        inputGuestInfo.sendKeys(info);
+    }
+
+    @When("The user clicks on the send message button")
+    public void theUserClicksOnTheSendMessageButton() {
+        SeleniumActions.click(ContactUsPage.sendMessageButton, driver);
+    }
+
+    @Then("User validates message is successfully sent")
+    public void userValidatesMessageIsSuccessfullySent() {
+        SeleniumActions.waitForVisibility(ContactUsPage.successMessageDisplay, driver);
+        WebElement contactUsPage = driver.findElement(ContactUsPage.successMessageDisplay);
```

Comment:
rename this element, its not a contactUsPage

Reply:
Done